packages/itwinui-css/src/utils/icon/classes.scss

```diff
@@ -1,23 +1,26 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import './index';
+@use './icon-component.scss' as *;
 
-[data-iui-icon-color='default'] {
-  @include iui-icon-color;
-}
-
-[data-iui-icon-color='informational'] {
-  @include iui-icon-color(informational);
+.iui-svg-icon,
+[data-iui-icon-color] {
```

Comment:
Is there a reason why having `data-iui-icon-color` without `.iui-svg-icon` is still treated like an icon? I believe we should not change this logic to prevent breaking changes, but don't know why it was like this from before.

Reply:
yes it's to prevent breaking changes, and i believe it was done because it's already used in expandable-block (including in react)